packages/instrumentor/plugins/functionHooks.ts

```diff
@@ -46,8 +33,8 @@ export function functionHooks(filepath: string): () => PluginTarget {
 					if (path.node.params.every((param) => babel.isIdentifier(param))) {
 						const target = targetPath(path);
 						const matchedHooks = hookManager.matchingHooks(target, filepath);
-						if (applyHooks(target, path.node, matchedHooks)) {
-							logHooks(matchedHooks);
+						if (applyHooks(filepath, target, path.node, matchedHooks)) {
```

Comment:
If `filepath` is required in `applyHooks` now, the calculation of `matchedHooks` could be moved into that method as well.

Reply:
Nice! Refactored!